server/storage/consistent_store.go

```diff
@@ -0,0 +1,69 @@
+package storage
+
+import (
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/docker/go/canonical/json"
+	"github.com/docker/notary"
+	"github.com/docker/notary/tuf/data"
+)
+
+// ConsistentMetaStorage wraps a MetaStore in order to walk the TUF tree for GetCurrent in a consistent manner
```

Comment:
What is meant by 'Consistent' here? I was expecting it to mean 'lookup by hash' in the sense of 'consisent snapshots'. Is it something else?